code/__defines/apc.dm

```diff
@@ -0,0 +1,62 @@
+//update_state
```

Comment:
feels like these should be defined at the top of the APC.dm and later undefined, but this is also fine tbh

Reply:
Done.